Marlin/temperature.cpp

```diff
@@ -441,10 +441,9 @@ inline void _temp_error(int e, const char *msg1, const char *msg2) {
   if (!IsStopped()) {
     SERIAL_ERROR_START;
     if (e >= 0) SERIAL_ERRORLN((int)e);
-    serialprintPGM(msg1);
-    MYSERIAL.write('\n');
+    SERIAL_ERRORLNPGM(msg1);
     #ifdef ULTRA_LCD
-      lcd_setalertstatuspgm(msg2);
+      LCD_ALERTMESSAGEPGM(msg2);
```

Comment:
These macros wrap the argument in `PSTR()` but the input is already PSTR, so this won't work.